resources/views/templates/watch/header.blade.php

```diff
@@ -0,0 +1,160 @@
+                            </a>
+                        </div>
+                    </div>
+                    <!-- /LOGO -->
+
+                    <!-- SEARCH BAR -->
+                    <div class="col-md-6">
+                        <div class="header-search">
+                            {!! Form::open() !!}
+                            <select class="input-select">
+                                <option value="0">All Categories</option>
+                                @for ($i = 0; $i < count($menus); $i++) @if ($menus[$i]->parent_id===0)
+                                    <option value="1">{!! $menus[$i]->name !!}</option>
+                                    @endif
+                                    @endfor
+                            </select>
+                            {{Form::text('search',trans('messages.searchhere'),['class'=>'input'])}}
+                            {{ Form::submit(trans('messages.search'), ['class' => 'search-btn']) }}
+                            {!! Form::close() !!}
+
+                        </div>
+                    </div>
+                    <!-- /SEARCH BAR -->
+
+                    <!-- ACCOUNT -->
+                    <div class="col-md-3 clearfix">
+                        <div class="header-ctn">
+                            <!-- Cart -->
+                            <div class="dropdown">
+                                <a class="dropdown-toggle" data-toggle="dropdown" aria-expanded="true">
+                                    <i class="fa fa-shopping-cart"></i>
+                                    <span>@lang('messages.yourcart')</span>
+                                    <div class="qty">3</div>
+                                </a>
+                            </div>
+                            <!-- /Cart -->
+
+                            <!-- Menu Toogle -->
+                            <div class="menu-toggle">
+                                <a href="#">
+                                    <i class="fa fa-bars"></i>
+                                    <span>Menu</span>
+                                </a>
+                            </div>
+                            <!-- /Menu Toogle -->
+                        </div>
+                    </div>
+                    <!-- /ACCOUNT -->
+                </div>
+                <!-- row -->
+            </div>
+            <!-- container -->
+        </div>
+        <!-- /MAIN HEADER -->
+    </header>
+    <!-- /HEADER -->
+    <!-- NAVIGATION -->
+    <nav id="navigation">
+        <!-- container -->
+        <div class="container">
+            <!-- responsive-nav -->
+            <div id="responsive-nav">
+                <!-- NAV -->
+                <ul class="main-nav nav navbar-nav">
+                    <li class="active dropdown"><a href="#">@lang('messages.trangchu')</a>
+                    </li>
+                    @for ($i = 0; $i < count($menus); $i++) @if ($menus[$i]->parent_id === 0)
```

Comment:
đoạn code này làm gì

Reply:
đoạn này là em làm để làm cái navigate để nó hiện các thể loại mình chọn

Reply:
navigate của em sẽ có các thể loại chính và trong các thể loại có nhiều loại..
e cho vòng for chạy để lấy tất cả các thể loại ra và hiện lên navigate